### PR TITLE
Deprecate outdated carousel CSS and views on code.org

### PR DESCRIPTION
**Pull request by kelbyhawn**
Deprecates outdated CSS styles from various stylesheets around the https://code.org site. I also touched up some existing styles and deprecated the `carousel_pagination.haml` view since it's no longer used anywhere.

## Links
Jira ticket: [ACQ-1363](https://codedotorg.atlassian.net/browse/ACQ-1363)

## Testing story
Tested locally to make sure carousels still look as expected on key pages (see list below) and searched the codebase for class and ids for styles/stylesheets I'm deleting to make sure they aren't being used anywhere.
- http://localhost.code.org:3000/ai
  - This page has all three carousel types
- http://localhost.code.org:3000/teach
- http://localhost.code.org:3000/dance
- http://localhost.code.org:3000/minecraft
- http://localhost.code.org:3000/videos

### CODE REVIEW

File: pegasus/sites.v3/code.org/styles_min/040-page.css
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/styles_min/040-page.css:1]: I transferred some styles from the `design-system-pegasus.scss` sheet over to here.

File: apps/src/sites/code.org/pages/views/swiper.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ const swiperParams = {
       :host .swiper-pagination {
         position: relative;
         margin-top: 2rem;
+
+        .swiper-pagination-bullet {
+          margin-block: 0.5rem;
+        }
```
Comment (kelbyhawn) [inline apps/src/sites/code.org/pages/views/swiper.js:35]: This adds more vertical space between bullets to increase the tap area on mobile when there are 2+ rows of pagination.

File: pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss [hunk 2]
```diff
@@ -1075,14 +971,9 @@ form {
...
-        margin-right: 3rem;
-
-        @media screen and (max-width: $width-md) {
-          display: block;
-        }
 
-        @media screen and (max-width: $width-sm) {
-          margin-right: 0;
+        @media (min-width: $width-sm) {
+          margin-inline-end: 3rem;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss:976]: Formats image spacing correctly for RTL languages.

**Review by bethanyaconnor — APPROVED** [review]
🤩 love all this deleted code

**Pull request merged**